### DISCUSSION TITLE
Arandr and HerbstluftWM

### DISCUSSION THREAD

**Discussion by developer1105** [herbstluftwm/herbstluftwm#1461] [Q&A] [answered]
After running the "arandr" command to change screen resolution, things on the screen get smaller or bigger (depending on the change), but the already opened apps do not reflect the change.

They behave as if the resolution had not been changed. Is this a bug, or some config detail? This works automatically in i3.

**Comment by developer1105** [comment]
An example to test this is to open the Alacritty terminal at 960x540 (or a smaller resolution) and try to run bpytop on it. It will not run because the screen is too small. Then, with arandr, enlarge resolution to 1366x768. And then try to run bpytop again. It does not work.

I have tested this on i3, qtile, spectrwm, bspwm and awesomewm and I haven't had this issue. Up until now, I have only had this problem with herbstluftwm.

**Comment by t-wissmann** [comment, marked as answer]
When the monitor configuration changes, you can run
```
hc detect_monitors
```
to import the new monitor setup into hlwm. This can be done automatically with
```
hc set auto_detect_monitors on
```
(Panels still need to be restarted)

**Reply by developer1105** [reply]
Thanks. It worked!
